pkg/broker/broker.go

```diff
@@ -91,7 +91,7 @@ type MinibrokerClient interface {
 // Broker the parameters passed in.
 func NewBrokerFromOptions(o Options) (*Broker, error) {
 	klog.V(5).Infof("broker: creating a new broker with options %+v", o)
-	mb := minibroker.NewClient(o.ConfigNamespace, o.ServiceCatalogEnabledOnly)
+	mb := minibroker.NewClient(o.ConfigNamespace, o.ServiceCatalogEnabledOnly, o.ClusterDomain)
```

Comment:
It's getting to the point where passing in a struct there starts to look appealing (to ensure we don't pass arguments in the wrong order, by having named arguments).

Reply:
I follow the Law of Demeter. I won't pass the full object.

Reply:
I'm not asking you to pass in the full object; I'm asking for the arguments of the function to be a struct (that may or may not be the same struct).

Reply:
I see your point. Creating a struct for 3 parameters sounds overkill...